db/migrate/20170711235422_create_articles.rb

```diff
@@ -0,0 +1,14 @@
+ROM::SQL.migration do
+  change do
+    create_table :articles do
+      primary_key :id
+      column :slug, :text, null: false, unique: true
+      column :title, :text, null: false
+      column :body, :text, null: false
+      column :published, :boolean, default: false, null: false
```

Comment:
@timriley gah sorry, yeah it'd be `by_published(value)` doing `where(self[:published] => value)` which in this particular case doesn't look good